assets/sass/components/filter-card.component.scss

```diff
@@ -0,0 +1,106 @@
+@use "../_func.scss" as *;
+@use "card.module.scss" as *;
+
+:host {
+
+  --top-padding: #{rem(24)};
+  --right-padding: 1rem;
+  --bottom-padding: #{rem(24)};
+  --left-padding: 1rem;
+
+  &:has(.card__total) .card__body{
+
+    align-items: center;
+    display: flex;
+  }
+
+  &:before,
+  &:after {
+    display: none;
+  }
+
+  &.card--has-icon {
+    --bottom-padding: #{rem(16)};
+    border: none;
+  }
+
+  .icon {
+    font-size: rem(18)!important;
+    height: rem(18)!important;
+    margin: 0 0 0.5rem 0!important;
+    font-weight: 400!important;
+    display: block!important;
+  }
+    
+  .card__total {
+    margin-top: 0;
+    order: -1;
+  }
+}
+
+
+:host(:is(:hover,:focus,.hover)) {
+
+  outline: none;
+    
+  :is(.card__body,.card__footer) {
+    background: #EEEEEE;
+  }
+}
+
+:host-context(:is(:checked, .checked)),
+:host(:is(:active, .active)) {
+
+  outline: 2px solid var(--colour,var(--colour-primary));
+  outline-offset: -2px;
+
+  .card__body {
+    background: none;
+  }
+  .card__footer {
+    background: none;
+  }
+}
+
+
+::slotted(i) {
+  display: block !important;
+  font-size: rem(18)!important;
+  margin: 0 0 0.5rem 0!important;
+}
+
+
+// #region card with a flag 
+:host(.card--flag) {
+  position: relative;
+  
+  &:after {
+    content: "\f024";
```

Comment:
do we have to hard code the actual icon ?

Reply:
You can actually pull in values into CSS from data attributes, if you don't want to use props etc., I think this would work for font awesome values too: https://developer.mozilla.org/en-US/docs/Web/CSS/attr

Reply:
changed it so you can override with a CSS var